mmdet/apis/train.py

```diff
@@ -183,6 +186,14 @@ def build_optimizer(model, optimizer_cfg):
                 param_group['lr'] = base_lr * bias_lr_mult
                 if base_wd is not None:
                     param_group['weight_decay'] = base_wd * bias_decay_mult
+            if no_wd_in_dw:
+                module_name = name.replace('.weight', '').replace('.bias', '')
+                if module_name in named_modules:
+                    module = named_modules[module_name]
+                    # if this Conv2d is depthwise Conv2d
+                    if isinstance(module, torch.nn.Conv2d) and \
+                            module.in_channels == module.groups:
+                        param_group['weight_decay'] = 0
```

Comment:
Some optimizers do not accept the argument `weight_decay`. You may check if `base_wd` is not None and add an assertion in L159.